### ISSUE TITLE
Tab titles in overflow menu sometimes not correct

### ISSUE THREAD

**Issue by TAKeanice** [ViennaRSS/vienna-rss#1122]
The titles of tabs in the overflow menu are sometimes just labelled with the website name, instead of the title of the already loaded page.
![vienna_tabbar_bug](https://user-images.githubusercontent.com/8397407/38766301-2af967de-3fd0-11e8-813a-abe3af02f985.png)

**Comment by barijaona** [comment]
Yeah, the titles should be updated whenever Vienna receives the title of the current web page, but they are only updated on opening or closing a tab.

**Comment by TAKeanice** [comment]
I especially meant the title of the last tab in the overflow menu. It does not get updated even when the page has loaded for the first time, it keeps showing the URL of the page instead of the title.

**Comment by TAKeanice** [comment]
Fixing this is part of the browser project!

**Comment by TAKeanice** [comment]
This is still true, I might investigate it during the browser rework
